### ISSUE TITLE
[Bug]: Home: Views not loading

### ISSUE THREAD

**Issue by Flying-Ace** [astubenbord/paperless-mobile#399]
### What happened?

app worked on single users setup, but as soon as I configured a setup for multi user, the app could not display my views in HOME.  my current setup is every tag, document type, storage path have no owner so it can be shared. but the correspondents owner is same as login user as the app.

### Steps to reproduce

login. home screen has spinning wheel for views.

### Screenshots

![Screenshot_20240310-164628](https://github.com/astubenbord/paperless-mobile/assets/51868890/7df16f70-729e-4848-acab-54b72c1ac258)
![Screenshot_20240310-164543](https://github.com/astubenbord/paperless-mobile/assets/51868890/631c689e-f272-4482-aad1-ff771bef1b6b)
![2024-03-10_16-59-36](https://github.com/astubenbord/paperless-mobile/assets/51868890/7d65b266-1fae-4c12-bf76-fc734aad5d8d)

### Paperless-ng*x Version

2.61

### Paperless-Mobile Version

3.2.1 405

### Device

google pixel 7

### Operating System Version

android 14

### Device Locale

north america

### Selected Locale

English (US)

**Comment by Flying-Ace** [comment]
trial and error. it seems if the view has sort by= owner, and the document owner is (none), then the HOME: views crashes / has the spining wheel.  i changed the saved view template to sort by date and it works.  please confirm if this is true and add this fringe case for repair. thanks

thinking back on the error log: if it said SORT BY owner (instead of invalid argument owner) was not supported, it would have been easier to understand.  (d/t many item having an owner property)

**Comment by astubenbord** [comment]
Yeah, this seems to be an issue when using the current version of the app with newer versions. I'm currently by no means capable of keeping up with the speed at which new updates are released by the paperless-ngx team. I'm not even done with the multi-user update (API v4) which required a huge amount of changes to the app's architecture. Also I'm currently very busy and have hardly any time to work on the project.
